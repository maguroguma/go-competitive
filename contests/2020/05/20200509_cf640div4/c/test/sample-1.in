6
3 7
4 12
2 1000000000
7 97
1000000000 1000000000
2 1
