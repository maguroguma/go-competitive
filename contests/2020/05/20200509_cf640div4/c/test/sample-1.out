10
15
1999999999
113
1000000001
1
